docs/source/workflows/mcp/mcp-client.md

```diff
@@ -65,6 +65,61 @@ The function group supports filtering via the `include` and `exclude` parameters
 
 The function group can be directly referenced in the workflow configuration and provides all accessible tools from the MCP server to the workflow. Multiple function groups can be used in the same workflow to access tools from multiple MCP servers. See [Function Groups](../function-groups.md) for more information about function group capabilities.
 
+#### Configuration Options
+
+The `mcp_client` function group supports the following configuration options:
```

Comment:
Add a note here that the user can also get the complete list of config options via -
```
nat info components -t function_groups -q mcp_client
```